### PR TITLE
fix default hostname for chronos and spark

### PR DESCRIPTION
**Pull request by elingg**
@jsancio , @ConnorDoyle, please take a look ASAP, thanks!

### CODE REVIEW

File: repo/packages/C/chronos/0/config.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     "chronos/hostname": {
       "description": "The advertised hostname stored in ZooKeeper so another standby host can redirect to this elected leader (default = localhost)",
       "type": "string",
-      "default": "`curl http://169.254.169.254/latest/meta-data/public-hostname`"
+      "default": "$HOST"
```
Comment (ConnorDoyle) [inline repo/packages/C/chronos/0/config.json]: Nice one!

Comment (jdef) [inline repo/packages/C/chronos/0/config.json]: This looks pretty specific to aws. What's the priority for supporting other
clouds?

--sent from my phone

Comment (elingg) [inline repo/packages/C/chronos/0/config.json]: Actually, this is an env variable in Marathon so it should work consistently for different cloud platforms.

Comment (jdef) [inline repo/packages/C/chronos/0/config.json]: Yep, I mentally reversed the diff when I read it through the first time.
Sorry for the noise.

On Sat, Apr 25, 2015 at 1:33 PM, Elizabeth Lingg <notifications@github.com>
wrote:

> In repo/packages/C/chronos/0/config.json
> <https://github.com/mesosphere/universe/pull/91#discussion_r29102379>:
>
> > @@ -72,7 +72,7 @@
> >      "chronos/hostname": {
> >        "description": "The advertised hostname stored in ZooKeeper so another standby host can redirect to this elected leader (default = localhost)",
> >        "type": "string",
> > -      "default": "`curl http://169.254.169.254/latest/meta-data/public-hostname` <http://169.254.169.254/latest/meta-data/public-hostname>"
> > +      "default": "$HOST"
>
> Actually, this is an env variable in Marathon so it should work
> consistently for different cloud platforms.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/universe/pull/91/files#r29102379>.
>



-- 
James DeFelice
585.241.9488 (voice)
650.649.6071 (fax)

**Pull request merged**